EasyBusiness/src/components/Login/login.js

```diff
@@ -9,9 +9,27 @@ import Button from "../Button/button";
 const LoginForm = () => {
   const [user, setUser] = useState("");
   const [password, setPassword] = useState("");
+  const CheckUser = e => {
```

Comment:
If this function is not a component, please make sure the first letter is not capitalised.